### ISSUE TITLE
Why  is the perfomance of MBPA++ of your implementation much higher than the original results?

### ISSUE THREAD

**Issue by kyz20** [jojotenya/LAMOL#3]
only beacuse of your data format?

**Comment by Daikon-Sun** [comment]
We believe the data format is the same because we use the same datasets.
Notice our implementation of MBPA++ doesn't use the QA format.
We try to follow the paper as much as possible, except
1. As mentioned in MBPA++, they randomly sampled each dataset to the size of the smallest one. We also perform this sampling, but the outcome is (certainly) not the same.
2. Some hyperparameters are not the same. The most important ones are the local adaptation learning rate and the local adaptation steps. We tune our hyperparameters to optimize our implementation.

We don't really believe these are the reasons for the difference in performance.
Thus, to be honest, we don't really know why our performance is much better.
The code of MBPA++ is not released either.
That's the reason we included the scores of our implementation in our paper.
